python/llm/example/GPU/LLM-Finetuning/LoRA/deepspeed_zero3_config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "zero_optimization": {
+      "stage": 3,
+      "contiguous_gradients": true,
+      "overlap_comm": true,
+      "offload_optimizer": {"device": "cpu"}
```

Comment:
Do we need to add `offload_param`?

Reply:
The same memory usage was shown no matter with or without offload_param, and therefore it tends to be an Arc-platform related issue. To avoid potential issues, it is removed.